pkg/ovs/ovsctl/appctl.go

```diff
@@ -227,3 +228,49 @@ func (c *ovsCtlClient) GetDPFeatures() (map[DPFeature]bool, error) {
 	}
 	return features, nil
 }
+
+// DeleteDPInterface deletes OVS datapath interface, and it returns with no error if the interface does not exist.
+func (c *ovsCtlClient) DeleteDPInterface(name string) error {
+	cmd := fmt.Sprintf("dpctl/show ovs-system")
+	out, err := c.runAppCtl(cmd, true)
+	if err == nil {
+		return nil
+	}
+	scanner := bufio.NewScanner(strings.NewReader(string(out)))
+	scanner.Split(bufio.ScanLines)
+	ports := make([]int, 0)
+	for scanner.Scan() {
+		line := scanner.Text()
+		fields := strings.Split(line, ": ")
+		if len(fields) != 2 {
+			klog.InfoS("Unexpected output from dpif/show ovs-system", "line", line)
+			continue
+		}
+		nameStr := fields[1]
+		ifName := strings.Split(nameStr, "(internal)")[0]
+		if ifName != name {
```

Comment:
there is a space before `(internal)`, so the name would never match if it tries to remove an internal port.
```
port 2: antrea-gw0 (internal)
```

Reply:
Done